### PR TITLE
cEP-0000.md: Allow cEP-# instead of cEP-####

### PR DESCRIPTION
**Pull request by arnab1896**
This allows authors to write cEPs without the need
the need to add un-necessary leading zeroes.

Fixes https://github.com/coala/cEPs/issues/16

### CODE REVIEW

**Comment by meetmangukiya** [comment]
Also rename the existing cEPs to fit in well ? or another issue perhaps

**Comment by sims1253** [comment]
Don't change the existing cEP names. Just the Rule.

**Comment by sims1253** [comment]
and this is not a Bugfix so please use the Closes keyword instead of Fixes in your commit message

**Review by damngamerz — APPROVED** [review]
ack 712afd71c27ab5922cca3f07c98ccbdf2fd1d8a7
seems fine to me now!!

File: cEP-0000.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ coala Enhancement Proposals
 
 |Metadata|                                   |
 |--------|-----------------------------------|
-|cEP     |0000                               |
+|cEP     |0                                  |
```
Comment (Mixih) [inline cEP-0000.md]: Why did you change this?

Comment (arnab1896) [reply cEP-0000.md]: my bad. will revert

File: cEP-0000.md [hunk 2]
```diff
@@ -29,8 +29,11 @@ Spelling
 As we all know, coala is written with a lower case c. The proper spelling of a
 cEP is `cEP-[0-9]{4}([a-z])?`, so `cEP-####` or `cEP-####x`.
 
-Invalid ways to express cEP specifications include `CEP-####`, `cEP####`,
-`cep-####`, and `cEP-#`.
+A cEP may also be spelled as `cEP-#` , `cEP-##` or `cEP-###` in case the
+author wants to.
```
Comment (sims1253) [inline cEP-0000.md]: Add something like "leaving out teh zeros"

**Comment by arnab1896** [comment]
@sims1253 ; done. :smile:

**Comment by Udayan12167** [comment]
Add version change as well maybe?

**Comment by arnab1896** [comment]
@Udayan12167 , what do you mean? :smile:

**Comment by Mariatta** [comment]
@arnab1896 cEP version number should be increased, eg 2.2

File: cEP-0000.md [hunk 3]
```diff
@@ -29,8 +29,12 @@ Spelling
 As we all know, coala is written with a lower case c. The proper spelling of a
 cEP is `cEP-[0-9]{4}([a-z])?`, so `cEP-####` or `cEP-####x`.
 
-Invalid ways to express cEP specifications include `CEP-####`, `cEP####`,
-`cep-####`, and `cEP-#`.
+<NOTE> Leaving out the zeroes :-
+A cEP may also be spelled as `cEP-#` , `cEP-##` or `cEP-###` in case the
+author wants to.
```
Comment (sils) [inline cEP-0000.md]: don't make this a note please, just add it to the previous sentence like it was always there.

Also don't forget to update the version of the cEP

**Comment by arnab1896** [comment]
@Mariatta , is the version correct now? Also, i have removed the word 'NOTE'.

File: cEP-0000.md [hunk 4]
```diff
@@ -29,8 +29,12 @@ Spelling
 As we all know, coala is written with a lower case c. The proper spelling of a
 cEP is `cEP-[0-9]{4}([a-z])?`, so `cEP-####` or `cEP-####x`.
 
-Invalid ways to express cEP specifications include `CEP-####`, `cEP####`,
-`cep-####`, and `cEP-#`.
+Leaving out the zeroes :-
```
Comment (Udayan12167) [inline cEP-0000.md]: Don't use a separate header so remove this.

File: cEP-0000.md [hunk 5]
```diff
@@ -29,8 +29,12 @@ Spelling
...
+A cEP may also be spelled as `cEP-#` , `cEP-##` or `cEP-###` in case the
```
Comment (Udayan12167) [inline cEP-0000.md]: Add this sentence to the previous sentence on line 30.

**Comment by arnab1896** [comment]
@Udayan12167 ; let me know if i have made any mistakes.

Cheers! :tada:

**Comment by Mixih** [comment]
@arnab1896 please don't ping the specific reviewers. After you make a change, it will automatically be added to the review queue.

File: cEP-0000.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ coala Enhancement Proposals
 |Metadata|                                   |
 |--------|-----------------------------------|
 |cEP     |0000                               |
-|Version |2.1                                |
+|Version |2.2                                |
```
Comment (AsnelChristian) [inline cEP-0000.md:7]: you should do this in a different commit

File: cEP-0000.md [hunk 7]
```diff
@@ -28,9 +28,11 @@ Spelling
 
 As we all know, coala is written with a lower case c. The proper spelling of a
 cEP is `cEP-[0-9]{4}([a-z])?`, so `cEP-####` or `cEP-####x`.
+A cEP may also be spelled as `cEP-#` , `cEP-##` or `cEP-###` in case the
+author wants to.
 
-Invalid ways to express cEP specifications include `CEP-####`, `cEP####`,
-`cep-####`, and `cEP-#`.
+Invalid ways to express cEP specifications include `CEP-####`, `cEP####` and
+`cep-####`.
```
Comment (AsnelChristian) [inline cEP-0000.md]: Cep, CeP, CEp

Comment (sils) [reply cEP-0000.md]: no need to list all.

File: cEP-0000.md (hunk 6 above)
Comment (sils) [reply cEP-0000.md:7]: @Asnelchristian why? The version change belongs to the rest IMO. I don't wanna make this harder for @arnab1896 because it doesn't matter too much so either is fine with me now.

**Comment by sils** [comment]
ack https://github.com/coala/cEPs/pull/63/commits/479c2fc4539b570b37c6398164f5f6f22b516a39 https://github.com/coala/cEPs/pull/63/commits/b83712d5531accc5188806cd602af0fab8de757f

I think we've had enough beurocracy on this PR...

**Comment by sils** [comment]
So we do need a second ack here to merge this!

**Comment by sils** [comment]
ack b83712d

**Comment by sils** [comment]
(From someone else, @underyx ?)

**Comment by underyx** [comment]
ack 479c2fc b83712d

**Comment by sils** [comment]
@rultor merge

**Comment by arnab1896** [comment]
did rultor successfully merge?

**Comment by sils** [comment]
@rultor merge
